python/doc/theory/reliability_sensitivity/sorm_approximation.rst

```diff
@@ -3,92 +3,91 @@
+
+Furthermore, we suppose that outside the sphere which tangents the limit state surface in the standard
+space, :math:`f_{\vect{U}}` is decreasing.
+
+The difference with FORM comes from the approximation of the limit state surface at the design point :math:`P^*` in the standard space: SORM approximates it by a quadratic surface that has the same main curvatures at the design point.
+Let  :math:`n` be the dimension of the random vector :math:`\vect{X}` and
+:math:`(\kappa_i)_{1 \leq i \leq n-1}` the :math:`(n-1)` main curvatures of the limit state function at the design point in the standard space.
+Several approximations are available,
+detailed here in the case where the origin of the standard
+space does not belong to the failure domain:
+
+**Breitung’s formula** is an asymptotic results: the
+usual formula used in the normal standard space, has been generalized
+in [lebrun2009b]_ to standard spaces where the
+distribution is spherical, with :math:`E` the marginal cumulative
+density function of the spherical distributions in the standard space:
 
 .. math::
   :label: PfSORM_B
 
-    P_{Breitung}^{generalized}  \stackrel{\beta\rightarrow\infty}{=} E(-\beta)\prod_{i=1}^{n-1}\frac{1}{\sqrt{1+\beta\kappa_i}}
+    P_{Breitung}^{generalized}  \stackrel{\beta\rightarrow\infty}{=} E(-\beta)\prod_{i=1}^{n-1}\frac{1}{\sqrt{1+\kappa_i^0}}
 
-where :math:`\Phi` is the cumulative distribution function of the
-standard 1D normal distribution.
 
--  Hohenbichler’s formula is an approximation of :eq:`PfSORM_B`:
+where :math:`\Phi` is the cumulative distribution function of the standard 1D normal
+distribution and  :math:`(\kappa_1^0, \dots, \kappa_d^0)` the main curvatures of the
+homothetic of the failure domain at distance 1 from the origin.
 
-   .. math::
-     :label: PfSORM_HB
+**Hohenbichler’s formula** is an approximation of :eq:`PfSORM_B`:
 
-        \displaystyle P_{Hohenbichler} = \Phi(-\beta_{HL}) \prod_{i=1}^{n-1} \left( 1+\frac{\phi(-\beta_{HL})}{\Phi(-\beta_{HL})}\kappa_i  \right)  ^{-1/2}
+.. math::
+    :label: PfSORM_HB
 
-   **This formula is valid only in the normal standard space and if**
-   :math:`\boldsymbol{\forall i, 1+\frac{\phi(-\beta_{HL})}{\Phi(-\beta_{HL})}\kappa_i > 0}`.
+       \displaystyle P_{Hohenbichler} = \Phi(-\beta) \prod_{i=1}^{n-1} \left( 1+\frac{\phi(-\beta)}{\beta \Phi(-\beta)}\kappa_i^0  \right)  ^{-1/2}
 
--  | Tvedt’s formula (Tvedt, 1988):
+Recording to the Mill's ratio, :math:`\frac{\phi(-\beta)}{\beta \Phi(-\beta)}` tends to 1 when :math:`\beta` tends to :math:`+\infty`.
+This formula is valid **only** in the normal standard space and if:
 
-     .. math::
+.. math::
+
+    1+\frac{\phi(-\beta)}{\beta\Phi(-\beta)}\kappa_i^0 > 0
+
+for any :math:`i`.
+
+**Tvedt’s formula** (Tvedt, 1988):
+
+.. math::
        :label: PfSORM_T
 
             \left\{
```

Comment:
Please split the equation into 4 different simpler pieces, because the layout is not as aesthetic as it could be at this point:

![image](https://user-images.githubusercontent.com/31351465/229276019-265abaef-b5dd-44e4-93b5-e8027041ff08.png)

This can be written:

![image](https://user-images.githubusercontent.com/31351465/229276261-20182c38-6c51-4b86-a5e8-a486497237cc.png)

Reply:
ok It is better.